src/casl/guards/policies.guard.ts

```diff
@@ -18,7 +18,8 @@ export class PoliciesGuard implements CanActivate {
         context.getHandler(),
       ) || [];
 
-    const { user } = context.switchToHttp().getRequest();
+    const req = context.switchToHttp().getRequest();
```

Comment:
This change might not be needed because it was one-liner previously and now it's two for exactly same thing.